app/src/main/java/org/simple/clinic/analytics/DebugAnalyticsReporter.kt

```diff
@@ -1,18 +1,16 @@
 package org.simple.clinic.analytics
 
-import timber.log.Timber
-
 class DebugAnalyticsReporter : AnalyticsReporter {
 
   override fun setUserIdentity(id: String) {
-    Timber.tag("Analytics").d("User ID: $id")
+    // No-op.
```

Comment:
Instead of No-Opping them, could you instead not add the `DebugAnalyticsReporter` to the `Analytics` class? This will allow us to add it again when we need to test new analytics events.

Reply:
That sounds much better. I'll do this.